index.js

```diff
@@ -21,6 +23,30 @@ const transformBabel = src => {
   return result;
 };
 
+
+const transformTs = (src, path) => {
+  const {config} = tsconfig.loadSync(process.cwd());
```

Comment:
maybe just old good `require` would be sufficient here

`const {compilerOptions} = require(path.resolve(process.cwd(),'tsconfig.json')`